Resources/Prototypes/Magic/mindswap_spell.yml

```diff
@@ -1,21 +1,21 @@
 - type: entity
+  parent: BaseEntitySpellAction
   id: ActionMindSwap
   name: Mind Swap
   description: Exchange bodies with another person!
   components:
-  - type: EntityTargetAction
+  - type: Action
     useDelay: 300
-    itemIconStyle: BigAction
-    whitelist:
-      components:
-      - Body
-      - MindContainer
-    canTargetSelf: false
-    interactOnMiss: false
     sound: !type:SoundPathSpecifier
       path: /Audio/Magic/staff_animation.ogg
     icon:
       sprite: Mobs/Species/Human/organs.rsi
       state: brain
+  - type: EntityTargetAction
+    whitelist:
+      requireAll: true
```

Comment:
notice how it didnt have requireAll before so wizards could mindswap into ANY mind container, even pais and shit